packages/camera/example/android/gradlew

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env bash
```

Comment:
Don't check this into the flutter/plugins repo. Add to .gitignore.

Reply:
Done